### ISSUE TITLE
netmask_bits() returns 32 for all zero mask

### ISSUE THREAD

**Issue by adamhj** [drkjam/netaddr#122]
a simple bug:
>>> z=IPAddress('0.0.0.0')
>>> z.netmask_bits()
32

**Comment by nnathan** [comment]
Just some context, it appears any valid subnet mask shows the correct amount of subnet bits. So in the case of `0.0.0.0` it should have the value 0.

```python
In [1]: from netaddr import IPAddress

In [2]: for ip in [ '0.0.0.0', '128.0.0.0', '192.0.0.0', '255.0.0.0', '255.128.0.0' ]:
    print ip + ": " + str(IPAddress(ip).netmask_bits())
   ...:     
0.0.0.0: 32
128.0.0.0: 1
192.0.0.0: 2
255.0.0.0: 8
255.128.0.0: 9
```

**Comment by drkjam** [comment]
Fixed by pull request #129. Thanks @nnathan
